README.Rmd

```diff
@@ -159,3 +159,39 @@ By contributing to this project, you agree to abide by its terms.
 ```{r message=FALSE, warning=FALSE}
 citation("simulist")
 ```
+
+## Related projects
+
+This project has some overlap with other R packages. Here we list these packages and provide a table of features and attributes that are present for each package to help decide which package is appropriate for each use-case.
+
+In some cases the packages are dedicated to simulating line list and other epidemiological data (e.g. {simulist}), in others the line list simulation is one part of a wider R package (e.g. {EpiNow}). 
+
+- [`{LLsim}`](https://github.com/jrcpulliam/LLsim) simulates line list data using a stochastic SIR model with a fixed population with observation and reporting delays.
+- [`{simulacr}`](https://github.com/reconhub/simulacr) uses a branching process model to simulate cases and contacts for an outbreak. It simulates transmission of infections using other epidemiological R packages (`{epicontacts}` and `{distcrete}`) to parameterise and plot simulated data.
+- [`{epidict}`](https://github.com/R4EPI/epidict) is a package for data dictionaries for outbreaks and surveys. It can generate line list data using an outbreak data dictionary (`gen_data()`). Currently MSF outbreak data dictionaries for Acute Jaundice Syndrome, Cholera, Measles and Meningitis are available.
+- [`{EpiNow}`](https://github.com/epiforecasts/EpiNow) - a now deprecated R package - includes the `simulate_cases()` and `generate_pseudo_linelist()` functions for generating line list data.
```

Comment:
> Is the list's justification to compare simulist to existing projects to showcase what simulist can do that others can't or to act as a resource so users can access other packages for those cases where using simulist is not an option?

I would say definitely the latter. As an open source project, {simulist} doesn't want to claim a monopoly of functionality over every other package that is related or overlaps with {simulist}. This is just a guide in case the user wants to work out which tools can help them to accomplish their task. As part of this the feature table will highlight what {simulist} can do that the other packages can't (or vice versa) but it is not to show what other packages cannot do.

> most importantly it's a deprecated package- should we even include it in this list?

For now I think it's best to include, however, if over time we think that we should remove deprecated packages from the list then we can do. Arguably, packages that are not maintained and haven't been developed in several years could also be termed deprecated, it is just the maintainer has not explicitly tagged the package as such.